### PR TITLE
use default deployed ruby for legacy branch and add to test matrix

### PR DESCRIPTION
**Pull request by peetucket**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,5 +10,6 @@ env:
     - ROBOT_ENVIRONMENT=test
 
 rvm:
-  - 2.3.7 # deployed
+  - 2.3.7
```
Comment (jcoyne) [inline .travis.yml]: Why keep these around if we aren't deploying on these versions?

Comment (peetucket) [reply .travis.yml]: yeah, i'll ditch 2.3.7

File: .travis.yml [hunk 2]
```diff
@@ -10,5 +10,5 @@ env:
     - ROBOT_ENVIRONMENT=test
 
 rvm:
-  - 2.3.7 # deployed
   - 2.4.4
```
Comment (jcoyne) [inline .travis.yml]: Can we get rid of 2.4 also?

Comment (peetucket) [reply .travis.yml]: oops, missed before I merged, sorry

i think the answer would have yes, though

**Pull request merged**